### PR TITLE
Added the formatDatetime and gravatarURL functions

### PR DESCRIPTION
**Pull request by TTA777**
Implementations for formatDatetime and gravatarURL

### CODE REVIEW

File: minitwit.go [hunk 1]
```diff
@@ -53,10 +57,17 @@ func getUserID(username string) []interface{} {
 }
 
 // formatDatetime formats a timestamp for display.
-func formatDatetime(timestamp int64) string { return "" }
+func formatDatetime(timestamp int64) string {
+	var timeObject = time.Unix(timestamp, 0)
```
Comment (svopper) [inline minitwit.go:61]: I suggest that we use short hand syntax for variable declaration to be consistent across codebase. 
```golang
    timeObject := ...
```

**Review by svopper — CHANGES_REQUESTED** [review]
Added a comment about variable declarations.

File: minitwit.go (hunk 1 above)
Comment (TTA777) [reply minitwit.go]: Hadn't realized that was a thing. Just updated the code to use that instead

Comment (svopper) [reply minitwit.go]: No problem. Approving.

**Review by svopper — APPROVED** [review]

**Comment by svopper** [comment]
And remember to delete the branch :)

**Pull request merged**